en/services/timesync_v1.md

```diff
@@ -0,0 +1,45 @@
+# Time Synchronization Protocol v1
+
+This protocol is used to estimate the time offsets between system clocks on two MAVLink systems, in a two-system MAVLink network.
+This is useful because it allows ....
+
+The protocol uses just one message [TIMESYNC](#TIMESYNC), which has two `int64_t` fields, `tc1` and `ts1`. 
+The message is sent out with a nanosecond timestamp in `ts1` (current unix timestamp * 1e9).
```

Comment:
Do existing implementations use UNIX timestamps? It appears not.

Reply:
What makes you think they aren't?

Reply:
[The PX4 implementation](https://github.com/PX4/PX4-Autopilot/blob/0ea347a5c9f3f419a01f6b8e7f7952e7455b9aa7/src/modules/mavlink/mavlink_timesync.cpp#L52) uses `hrt_absolute_time()`. I'm under the impression that `hrt_absolute_time()` returns a timestamp from a hardware timer, not a unix timestamp.

Reply:
@julianoes do you know if @obicons is correct ? I.e. I am absolutely sure the method returns a timestamp from the hardware timer but I am not sure whether or not there is anything else that then synchronizes that to an absolute time. I highly suspect that even if it is possible, you can't rely on it.

As I understand it timesync is used mostly to provide accurate relative time sync - ie it doesn't matter that they aren't at the same time as long as the offset in values provided by both is known. 
- Is that correct?
- Are there use cases where the values _need_ to be a timestamp against a known reference like unix epoch?

If not, I'd suggest we change the message to make it seconds since boot when I add the extensions for sysid/component id.

Reply:
Other messages allow timestamps in both formats (i.e., epoch and system boot.) Perhaps allow either in this message, and update the documentation?

Reply:
Fair enough. The way to progress this is to test it in a message update: https://github.com/mavlink/mavlink/pull/1878

Come back here when we work out the appetite for change.